build.gradle

```diff
@@ -1,10 +1,10 @@
 buildscript {
     repositories {
-		mavenCentral()
```

Comment:
https://blog.bintray.com/2015/02/09/android-studio-migration-from-maven-central-to-jcenter/